algorithms/polish/stemmer.tt

```diff
@@ -0,0 +1,107 @@
+[% header('Polish stemming algorithm') %]
+
+<h2>Links to resources</h2>
+
+<ul>
+[% algorithm_lis('polish', 'Polish') %]
+<li><a href="https://github.com/Tutanchamon/pl_stemmer">Another Polish stemmer, written in Python</a></li>
+</ul>
+
+<p>
+Here is a sample of Polish vocabulary, with the stemmed forms that will be
+generated by this algorithm:
+</p>
+
+[% algorithm_vocab([20, 'aktualizacja', 'kwiecień']) %]
+
+<h2>The stemming algorithm</h2>
+
+<p>
+The Polish alphabet includes the following additional letters:
+Ą, Ć, Ę, Ł, Ń, Ó, Ś, Ź, and Ż.
```

Comment:
The additional letters are shown in lowercase in the other stemmer descriptions - I think that's better as the algorithms assume their input has been folded to lowercase.

I notice _Ż_ is listed here but doesn't seem to be defined for the algorithm.  Presumably that's just because it doesn't need special treatment?